src/Bindings/RabbitMQClientBuilder.cs

```diff
@@ -31,14 +31,9 @@ private IModel CreateModelFromAttribute(RabbitMQAttribute attribute)
             }
 
             string resolvedConnectionString = Utility.FirstOrDefault(attribute.ConnectionStringSetting, _options.Value.ConnectionString);
-            string resolvedHostName = Utility.FirstOrDefault(attribute.HostName, _options.Value.HostName);
-            string resolvedUserName = Utility.FirstOrDefault(attribute.UserName, _options.Value.UserName);
-            string resolvedPassword = Utility.FirstOrDefault(attribute.Password, _options.Value.Password);
-            int resolvedPort = Utility.FirstOrDefault(attribute.Port, _options.Value.Port);
-            bool resolvedEnableSsl = Utility.FirstOrDefault(attribute.EnableSsl, _options.Value.EnableSsl);
-            bool resolvedSkipCertificateValidation = Utility.FirstOrDefault(attribute.SkipCertificateValidation, _options.Value.SkipCertificateValidation);
-
-            IRabbitMQService service = _configProvider.GetService(resolvedConnectionString, resolvedHostName, resolvedUserName, resolvedPassword, resolvedPort, resolvedEnableSsl, resolvedSkipCertificateValidation);
+            bool resolvedDisableCertificateValidation = Utility.FirstOrDefault(attribute.DisableCertificateValidation, _options.Value.DisableCertificateValidation);
+
+            IRabbitMQService service = _configProvider.GetService(resolvedConnectionString, resolvedDisableCertificateValidation);
 

```

Comment:
Removed connection string elements: `HostName`, `UserName`, `Password`, `Port`, and `EnableSsl` from both trigger and output binding attributes. Renamed `SkipCertificateValidation` to `DisableCertificateValidation`. The `SkipCertificateValidation` property was added recently and was not released to users.